### PR TITLE
Add example of disabling fleet logging feature

### PR DESCRIPTION
**Pull request by JeffLuoo**
Add example of disabling fleet logging feature

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
gkehub: added example of disabling fleet logging feature (beta)
```

### CODE REVIEW

**Review by courageJ — APPROVED** [review]

**Comment by JeffLuoo** [comment]
@slevenick Could you please help take a look of the PR? Thanks!

**Review by slevenick — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/examples/disable_fleet_observability.tf.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+resource "google_gke_hub_feature" "feature" {
+  project = "jeffluoo-anthos-dev"
```
[the remark this answers is not in the document]
Comment (JeffLuoo) [reply mmv1/templates/terraform/examples/disable_fleet_observability.tf.erb]: Updated to remove the project to match with existing examples.

**Comment by JeffLuoo** [comment]
I didn't touch the test `TestAccComputeFirewallPolicyRule_multipleRules`, do you happen to know why does it fail on my PR? @slevenick Thanks!

**Review by slevenick — CHANGES_REQUESTED** [review]
Looks like this test fails. Are you able to run it yourself?

**Comment by slevenick** [comment]
Oh, no I mean the test that you added. TestAccGKEHubFeature_gkehubFeatureFleetObservability seems to fail now. It's skipped in VCR so you won't see it failing, but I ran it manually

**Review by slevenick — CHANGES_REQUESTED** [review]
See comment, TestAccGKEHubFeature_gkehubFeatureFleetObservability is failing

**Comment by JeffLuoo** [comment]
May I ask for how do you run the test locally? I initially run the same config with tf and it worked so I just copied the existing test. I then followed https://googlecloudplatform.github.io/magic-modules/get-started/run-provider-tests/ to run the test locally but ran into bunch of errors like I don't have the permission to update the test project. I used
```
TF_LOG=TRACE make testacc TEST=./google-beta TESTARGS='-run=TestAccGKEHubFeature_gkehubFeatureFleetObservability'
```
to run the test locally.

**Comment by slevenick** [comment]
> 

Yeah, that command works to run the test locally. You need to do some environment setup first, like setting up your credentials and setting up a project with the correct APIs enabled.

**Comment by JeffLuoo** [comment]
What's the error you're seeing? When I ran the test locally I'm seeing either `ALREDY_EXIST` or `NOT_FOUND` error. But using the same terraform config and use the `terraform apply` doesn't return any error.

cc: @sahsagar-google

**Comment by slevenick** [comment]
The error:

```

          # google_gke_hub_feature.feature will be updated in-place
          ~ resource "google_gke_hub_feature" "feature" {
                create_time    = "2023-06-22T17:54:53.927694691Z"
                id             = "projects/tf-test-hub/locations/global/features/fleetobservability"
                labels         = {}
                location       = "global"
                name           = "fleetobservability"
                project        = "tf-test-hub"
                resource_state = [
                    {
                        has_resources = false
                        state         = "ACTIVE"
                    },
                ]
                state          = []
                update_time    = "2023-06-22T17:56:18.856728173Z"
              ~ spec {
                  ~ fleetobservability {
                      ~ logging_config {
                          + default_config {
                              + mode = "MODE_UNSPECIFIED"
                            }
                          + fleet_scope_logs_config {
                              + mode = "MODE_UNSPECIFIED"
                            }
                        }
                    }
                }
            }
```

**Comment by JeffLuoo** [comment]
Thank you for the update! @slevenick but I don't see any error messages from your comment.

**Comment by slevenick** [comment]
> Thank you for the update! @slevenick but I don't see any error messages from your comment.

Ah, sorry it's:
```
=== CONT  TestAccGKEHubFeature_gkehubFeatureFleetObservability
    vcr_utils.go:148: Step 7/8 error: After applying this test step, the plan was not empty.
        stdout:
        An execution plan has been generated and is shown below.
        Resource actions are indicated with the following symbols:
          ~ update in-place
        Terraform will perform the following actions:
``` 
with the above config after that.

**Pull request closed without merging**